### PR TITLE
Fix usage of backend instead of client in pytest

### PR DESCRIPTION
**Pull request by fbeutin-ledger**

### CODE REVIEW

File: template/conftest.py [hunk 1]
```diff
@@ -7,8 +7,10 @@
 
 
 # This variable is needed for Speculos only (physical tests need the application to be already installed)
+# Adapt this path to your 'tests/elfs' directory
 APPS_DIRECTORY = (Path(__file__).parent.parent / "elfs").resolve()
 
+# Adapt this path to the APPNAME in your Makefile
```
Comment (lpascal-ledger) [inline template/conftest.py:13]: Not really the APPNAME of the Makefile, but rather the `<name>_<device>.elf` used to store ELF binaries.

**Review by lpascal-ledger — APPROVED** [review]

**Review by lpascal-ledger — APPROVED** [review]

**Comment by lpascal-ledger** [comment]
As a template bug fix, this should be deployed ASAP, that's a good candidate for a `0.9.1`

**Review by xchapron-ledger — COMMENTED** [review]
I think we should update `ragger/tests/conftest.py` in this Pr too?

**Comment by fbeutin-ledger** [comment]
> I think we should update `ragger/tests/conftest.py` in this Pr too?

I suggest updating it next time we add tests in ragger, otherwise it's a bit of work for no gain

**Review by xchapron-ledger — APPROVED** [review]

**Pull request merged**